nextcord/client.py

```diff
@@ -2023,33 +2039,171 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         """
         return self._connection.get_application_command(command_id)
 
-    def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+    def __actual_get_application_command_from_signature(
```

Comment:
Why was this split from `get_application_command_from_signature`? Seems like an unnecessary change since this is only called once

Reply:
Yes but Ooliver wanted this to be a non-breaking change and since the signature was changed, I needed to use `*args, **kwargs` and do all the parsing.